understanding/page-titled.html

```diff
@@ -58,121 +56,94 @@
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 2.4.2:</span>Page Titled (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 2.4.2:</span>ページタイトル (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-web-page">Web pages</a> have titles that describe topic or purpose.
+                     <a href="#dfn-web-page">ウェブページ</a>には、主題又は目的を説明したタイトルがある。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Each web page has a meaningful title.</dd>
+                  <dt>目標</dt>
+                  <dd>各ウェブページに、意味のあるタイトルを付ける。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide a descriptive page title using appropriate technology.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>適切な技術を用いて、説明的なページタイトルを付ける。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Page titles help users identify and distinguish different pages.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>ページタイトルによって、利用者が異なるページを識別、及び区別するのに役立つ。</dd> 
```

Comment:
「ページタイトルによって…役立つ」というのは変なような。単に「は」で良いように思いましたがどうでしょうか。

```suggestion
                  <dd>ページタイトルは、利用者が異なるページを識別、及び区別するのに役立つ。</dd> 
```